blocks/parser.js

```diff
@@ -37,16 +39,14 @@ export function getBlockAttributes( blockNode, blockSettings ) {
  * @return {Array}          Block list
  */
 export default function parse( content ) {
-	return grammarParse( content ).reduce( ( memo, blockNode ) => {
+	return grammarParse( content ).map( ( blockNode ) => {
 		const settings = getBlockSettings( blockNode.blockType );
```

Comment:
Yeah, I'm thinking more an more that the distinction between `blocks` and `editor` is superficial. We could merge everything IMO. It can as two different global variables, but the module separation is not really important.